<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelUTSetupTimebase</class>
 <widget class="QWidget" name="PanelUTSetupTimebase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutTimebase">
     <item>
      <widget class="QGroupBox" name="delayGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>延时:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="layoutDelayTime">
          <item>
           <widget class="ConfigRadioButton" name="delayPreferTime">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="delayTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> us</string>
            </property>
            <property name="maximum">
             <double>99999.990000000005239</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="layoutDelayDistance">
          <item>
           <widget class="ConfigRadioButton" name="delayPreferDistance">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="delayDistance">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="maximum">
             <double>99999.990000000005239</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="rangeGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>声程:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="layoutRangeTime">
          <item>
           <widget class="ConfigRadioButton" name="rangePreferTime">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rangeTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> us</string>
            </property>
            <property name="maximum">
             <double>99999.990000000005239</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutRangeDistance">
          <item>
           <widget class="ConfigRadioButton" name="rangePreferDistance">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rangeDistance">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="maximum">
             <double>99999.990000000005239</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="materialGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>材料声速</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="changeMaterial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>更改材料</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMaterialName">
        <property name="text">
         <string>名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="materialName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMaterialSpeed">
        <property name="text">
         <string>速度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ConfigSpinBox" name="materialSpeed">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>mm/s</string>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigRadioButton</class>
   <extends>QRadioButton</extends>
   <header location="global">ConfigControl/ConfigRadioButton.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">ConfigControl/ConfigSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
